package.json

```diff
@@ -28,6 +29,23 @@
     "lint": "tslint -c tslint.full.json --project tsconfig.json --type-check --exclude \"examples/**/*\" --exclude \"**/node_modules/**\" --exclude \"**/*.d.ts\" \"**/*.ts\"",
     "lint:fix": "npm run lint -- --fix",
     "test": "mocha --opts test/mocha.opts.root \"packages/*/test/**/*.ts\"",
-    "posttest": "npm run lint"
+    "posttest": "npm run lint",
+    "coverage": "nyc npm test"
```

Comment:
Ultimately, we want to run `nyc` as part of `npm test`, so that we can report coverage data from Travis CI to Coveralls. See https://github.com/strongloop/loopback/pull/3165 for an example.

Having said that, I am ok to leave Coveralls integration for later.